node/src/components/consensus/era_supervisor.rs

```diff
@@ -1029,12 +1029,12 @@ impl EraSupervisor {
                 let era = self.open_eras.get_mut(&era_id).unwrap();
                 era.add_accusations(&equivocators);
                 era.add_accusations(value.accusations());
+
                 // If this is the era's last block, it contains rewards. Everyone who is accused in
                 // the block or seen as equivocating via the consensus protocol gets faulty.
                 let report = terminal_block_data.map(|tbd| EraReport {
```

Comment:
We include them in the `EraEnd` struct in the block. Here we only construct a `FinalizedBlock`, which is an internal node type, it's not exposed anywhere. Rewards used to be reported here, because they were being calculated by consensus, but now that they are calculated elsewhere, it makes sense to only include them in the final result, which is the `Block` struct.